### PR TITLE
Fix  google sheets + tests

### PR DESCRIPTION
**Pull request by madchap**
Addresses https://github.com/DefectDojo/django-DefectDojo/issues/3742 and https://github.com/DefectDojo/django-DefectDojo/issues/3744

### CODE REVIEW

File: dojo/db_migrations/0072_sheets_textfields.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.17 on 2021-01-30 07:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0071_product_type_enhancement'),
```
Comment (damiencarol) [inline dojo/db_migrations/0072_sheets_textfields.py:9]: we already passed this number.
```suggestion
        ('dojo', '0073_product_type_enhancement'),
```
Also rename the file

**Review by damiencarol — APPROVED** [review]

**Comment by damiencarol** [comment]
It seems that there is a file missing in the PR:
```
uwsgi_1         | ======================================================================
uwsgi_1         | ERROR: test_config_google_sheets (dojo.unittests.test_google_sheets_configuration.GoogleSheetsConfigTestApi)
uwsgi_1         | ----------------------------------------------------------------------
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/app/dojo/unittests/test_google_sheets_configuration.py", line 54, in test_config_google_sheets
uwsgi_1         |     with open('tests/defectdojo-sheets-localdev.json', 'rb') as f:
uwsgi_1         | FileNotFoundError: [Errno 2] No such file or directory: 'tests/defectdojo-sheets-localdev.json'
```

**Comment by madchap** [comment]
> It seems that there is a file missing in the PR:
> 
> ```
> uwsgi_1         | ======================================================================
> uwsgi_1         | ERROR: test_config_google_sheets (dojo.unittests.test_google_sheets_configuration.GoogleSheetsConfigTestApi)
> uwsgi_1         | ----------------------------------------------------------------------
> uwsgi_1         | Traceback (most recent call last):
> uwsgi_1         |   File "/app/dojo/unittests/test_google_sheets_configuration.py", line 54, in test_config_google_sheets
> uwsgi_1         |     with open('tests/defectdojo-sheets-localdev.json', 'rb') as f:
> uwsgi_1         | FileNotFoundError: [Errno 2] No such file or directory: 'tests/defectdojo-sheets-localdev.json'
> ```

Right, I was fighting the VCR, hence why the PR was still in Draft status when you reviewed it. Should be OK now, but waiting for tests to complete.

**Comment by madchap** [comment]
I could not make the VCR test work replay the stream with success if the json key file drifts too much. So that's the former key I used to test with, which obviously is now gone, account disabled, etc.. :man_shrugging: 

Wondering if we'll get an email saying we've pushed private creds..

If anyone has a better idea?

**Comment by valentijnscholten** [comment]
Find out which method performs the validation and mock it to always return True.

**Comment by valentijnscholten** [comment]
I tried to mock the validation method, but that skips some of the http requests breaking the test using the recorded/replayed request. So quickest solution is just to merge as-is. It looks weird, but the private key is not a real existing key, so no harm done.

**Pull request merged**